tests/test_home_page.py

```diff
@@ -0,0 +1,9 @@
+def test_home_page_dashboard():
+    # Open home page
+    # Check some value on the dashboard
```

Comment:
add code steps itself

Reply:
fixed in [035d1c4](https://github.com/edanilovets/aqa_hillel_1308/pull/1/commits/035d1c4fb8d1033078b566558d0c88a32fd52a78)